vnext/Microsoft.ReactNative/ReactApplication.idl

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT License.
 
 import "ReactNativeHost.idl";
+import "ReactRootView.idl";
```

Comment:
Why is it needed?

Reply:
old remnant from a previous iteration, removing